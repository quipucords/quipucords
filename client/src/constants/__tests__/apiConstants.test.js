import apiTypes, * as allApiTypes from '../apiConstants';

describe('ApiTypes', () => {
  it('Should have specific API properties', () => {
    expect(apiTypes).toMatchSnapshot('specific types');

    expect(allApiTypes).toMatchSnapshot('all api types');
  });
});
