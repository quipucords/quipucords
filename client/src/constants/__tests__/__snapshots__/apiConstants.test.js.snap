// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all exports 1`] = `
Object {
  "API_RESPONSE_CREDENTIALS_COUNT": "count",
  "API_RESPONSE_CREDENTIALS_RESULTS": "results",
  "API_RESPONSE_CREDENTIAL_CRED_TYPE": "cred_type",
  "API_RESPONSE_CREDENTIAL_ID": "id",
  "API_RESPONSE_CREDENTIAL_NAME": "name",
  "API_RESPONSE_JOBS_RESULTS": "results",
  "API_RESPONSE_SCANS_COUNT": "count",
  "API_RESPONSE_SCANS_RESULTS": "results",
  "API_RESPONSE_SOURCES_COUNT": "count",
  "API_RESPONSE_SOURCES_RESULTS": "results",
  "API_RESPONSE_SOURCE_CREDENTIALS": "credentials",
  "API_RESPONSE_SOURCE_CREDENTIALS_ID": "id",
  "API_RESPONSE_SOURCE_HOSTS": "hosts",
  "API_RESPONSE_SOURCE_ID": "id",
  "API_RESPONSE_SOURCE_NAME": "name",
  "API_RESPONSE_SOURCE_OPTIONS": "options",
  "API_RESPONSE_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
  "API_RESPONSE_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
  "API_RESPONSE_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
  "API_RESPONSE_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
  "API_RESPONSE_SOURCE_PORT": "port",
  "API_RESPONSE_SOURCE_SOURCE_TYPE": "source_type",
  "API_SUBMIT_SOURCE_CREDENTIALS": "credentials",
  "API_SUBMIT_SOURCE_HOSTS": "hosts",
  "API_SUBMIT_SOURCE_ID": "id",
  "API_SUBMIT_SOURCE_NAME": "name",
  "API_SUBMIT_SOURCE_OPTIONS": "options",
  "API_SUBMIT_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
  "API_SUBMIT_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
  "API_SUBMIT_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
  "API_SUBMIT_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
  "API_SUBMIT_SOURCE_PORT": "port",
  "API_SUBMIT_SOURCE_SOURCE_TYPE": "source_type",
  "apiTypes": Object {
    "API_RESPONSE_CREDENTIALS_COUNT": "count",
    "API_RESPONSE_CREDENTIALS_RESULTS": "results",
    "API_RESPONSE_CREDENTIAL_CRED_TYPE": "cred_type",
    "API_RESPONSE_CREDENTIAL_ID": "id",
    "API_RESPONSE_CREDENTIAL_NAME": "name",
    "API_RESPONSE_JOBS_RESULTS": "results",
    "API_RESPONSE_SCANS_COUNT": "count",
    "API_RESPONSE_SCANS_RESULTS": "results",
    "API_RESPONSE_SOURCES_COUNT": "count",
    "API_RESPONSE_SOURCES_RESULTS": "results",
    "API_RESPONSE_SOURCE_CREDENTIALS": "credentials",
    "API_RESPONSE_SOURCE_CREDENTIALS_ID": "id",
    "API_RESPONSE_SOURCE_HOSTS": "hosts",
    "API_RESPONSE_SOURCE_ID": "id",
    "API_RESPONSE_SOURCE_NAME": "name",
    "API_RESPONSE_SOURCE_OPTIONS": "options",
    "API_RESPONSE_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
    "API_RESPONSE_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
    "API_RESPONSE_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
    "API_RESPONSE_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
    "API_RESPONSE_SOURCE_PORT": "port",
    "API_RESPONSE_SOURCE_SOURCE_TYPE": "source_type",
    "API_SUBMIT_SOURCE_CREDENTIALS": "credentials",
    "API_SUBMIT_SOURCE_HOSTS": "hosts",
    "API_SUBMIT_SOURCE_ID": "id",
    "API_SUBMIT_SOURCE_NAME": "name",
    "API_SUBMIT_SOURCE_OPTIONS": "options",
    "API_SUBMIT_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
    "API_SUBMIT_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
    "API_SUBMIT_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
    "API_SUBMIT_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
    "API_SUBMIT_SOURCE_PORT": "port",
    "API_SUBMIT_SOURCE_SOURCE_TYPE": "source_type",
  },
  "default": Object {
    "API_RESPONSE_CREDENTIALS_COUNT": "count",
    "API_RESPONSE_CREDENTIALS_RESULTS": "results",
    "API_RESPONSE_CREDENTIAL_CRED_TYPE": "cred_type",
    "API_RESPONSE_CREDENTIAL_ID": "id",
    "API_RESPONSE_CREDENTIAL_NAME": "name",
    "API_RESPONSE_JOBS_RESULTS": "results",
    "API_RESPONSE_SCANS_COUNT": "count",
    "API_RESPONSE_SCANS_RESULTS": "results",
    "API_RESPONSE_SOURCES_COUNT": "count",
    "API_RESPONSE_SOURCES_RESULTS": "results",
    "API_RESPONSE_SOURCE_CREDENTIALS": "credentials",
    "API_RESPONSE_SOURCE_CREDENTIALS_ID": "id",
    "API_RESPONSE_SOURCE_HOSTS": "hosts",
    "API_RESPONSE_SOURCE_ID": "id",
    "API_RESPONSE_SOURCE_NAME": "name",
    "API_RESPONSE_SOURCE_OPTIONS": "options",
    "API_RESPONSE_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
    "API_RESPONSE_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
    "API_RESPONSE_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
    "API_RESPONSE_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
    "API_RESPONSE_SOURCE_PORT": "port",
    "API_RESPONSE_SOURCE_SOURCE_TYPE": "source_type",
    "API_SUBMIT_SOURCE_CREDENTIALS": "credentials",
    "API_SUBMIT_SOURCE_HOSTS": "hosts",
    "API_SUBMIT_SOURCE_ID": "id",
    "API_SUBMIT_SOURCE_NAME": "name",
    "API_SUBMIT_SOURCE_OPTIONS": "options",
    "API_SUBMIT_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
    "API_SUBMIT_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
    "API_SUBMIT_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
    "API_SUBMIT_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
    "API_SUBMIT_SOURCE_PORT": "port",
    "API_SUBMIT_SOURCE_SOURCE_TYPE": "source_type",
  },
}
`;

exports[`default export 1`] = `
Object {
  "API_RESPONSE_CREDENTIALS_COUNT": "count",
  "API_RESPONSE_CREDENTIALS_RESULTS": "results",
  "API_RESPONSE_CREDENTIAL_CRED_TYPE": "cred_type",
  "API_RESPONSE_CREDENTIAL_ID": "id",
  "API_RESPONSE_CREDENTIAL_NAME": "name",
  "API_RESPONSE_JOBS_RESULTS": "results",
  "API_RESPONSE_SCANS_COUNT": "count",
  "API_RESPONSE_SCANS_RESULTS": "results",
  "API_RESPONSE_SOURCES_COUNT": "count",
  "API_RESPONSE_SOURCES_RESULTS": "results",
  "API_RESPONSE_SOURCE_CREDENTIALS": "credentials",
  "API_RESPONSE_SOURCE_CREDENTIALS_ID": "id",
  "API_RESPONSE_SOURCE_HOSTS": "hosts",
  "API_RESPONSE_SOURCE_ID": "id",
  "API_RESPONSE_SOURCE_NAME": "name",
  "API_RESPONSE_SOURCE_OPTIONS": "options",
  "API_RESPONSE_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
  "API_RESPONSE_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
  "API_RESPONSE_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
  "API_RESPONSE_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
  "API_RESPONSE_SOURCE_PORT": "port",
  "API_RESPONSE_SOURCE_SOURCE_TYPE": "source_type",
  "API_SUBMIT_SOURCE_CREDENTIALS": "credentials",
  "API_SUBMIT_SOURCE_HOSTS": "hosts",
  "API_SUBMIT_SOURCE_ID": "id",
  "API_SUBMIT_SOURCE_NAME": "name",
  "API_SUBMIT_SOURCE_OPTIONS": "options",
  "API_SUBMIT_SOURCE_OPTIONS_DISABLE_SSL": "disable_ssl",
  "API_SUBMIT_SOURCE_OPTIONS_PARAMIKO": "use_paramiko",
  "API_SUBMIT_SOURCE_OPTIONS_SSL_CERT": "ssl_cert_verify",
  "API_SUBMIT_SOURCE_OPTIONS_SSL_PROTOCOL": "ssl_protocol",
  "API_SUBMIT_SOURCE_PORT": "port",
  "API_SUBMIT_SOURCE_SOURCE_TYPE": "source_type",
}
`;
